pkg/scm/git/git.go

```diff
@@ -18,6 +18,8 @@ import (
 // Git is an interface used by main STI code to extract/checkout git repositories
 type Git interface {
 	ValidCloneSpec(source string) bool
+	ValidCloneSpecRemoteOnly(source string) bool
+	MungeNoProtocolURL(source string, url *url.URL) bool
```

Comment:
MungeNoProtocolURL will be called by the pending origin changes.  There was duplicate logic in origin modules that have been consolidated in pkg/scm/git/git.go